src/hb-open-type-private.hh

```diff
@@ -909,7 +909,6 @@ struct UnsizedArrayOf
     return_trace (true);
   }
 
-  private:
```

Comment:
The only interesting part of this patch is this I guess, making `sanitize_shallow` public to use in `sbix` so we don't have to sanitize all the glyphs theirselves also.